### DISCUSSION TITLE
LOCAL is new reserved word?

### DISCUSSION THREAD

**Discussion by jluizsc** [FirebirdSQL/firebird#7977] [Q&A] [answered]
Hi Masters!!! 

Question: LOCAL is new reserved word?

-- BR

CREATE TABLE tstTable (
    LOCAL      INTEGER NOT NULL,
    OPERADOR   INTEGER NOT NULL,
    CODIGO     INTEGER NOT NULL,
    CAMPO      CHAR(15) NOT NULL,
);

/* Invalid token.
Dynamic SQL Error. SQL error code = -104.  Token unknown - line 1, column 9.  LOCAL. */

select LOCAL, OPERADOR, CODIGO, CAMPO from tstTable

/* Invalid token.
Dynamic SQL Error. SQL error code = -104. Token unknown - line 1, column 8. LOCAL.  */

I have many LOCAL in my DB

[]´s

**Comment by hvlad** [comment, marked as answer]
LOCAL is reserved since Firebird 4, see
https://firebirdsql.org/file/documentation/html/en/refdocs/fblangref40/firebird-40-language-reference.html#fblangref40-appx03-reskeywords

**Reply by jluizsc** [reply]
tks, I'm going to migrate from 2.5 and 3.0 directly to 5.0.... but... that's a problem that I have to solve!!
